docs/en/modules/net_info.md

```diff
@@ -0,0 +1,79 @@
+# net_info Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2017-01-21 | [smcl](http://blog.mclemon.io/esp8266-contributing-to-the-nodemcu-ecosystem) | [wolfgangr](https://github.com/wolfgangr/nodemcu-firmware) | [net_info.c](../../../app/modules/net_info.c)
+
+
+This module is a stub to collect common network diagnostic and analysis tools.
+
+##net_info.ping()
+ send ICMP ECHO_REQUEST to network hosts 
+ 
+**Synopsis:**:
```

Comment:
so this boils down to use 
####Synopsis
instead of 
\*\*Synopsis:\*\*

OK, no problem.
I had tried with 3 hashes ###
but still got shown in the index and I didn't want to clobber that with details too deep.

@devsaurus 
> Please stick to the common layout for API docs

Is there a common layout for API docs defined somewhere (outside of your brain, of course)?
I really would have preferred to start from some accepted template, 
if I only could have found one...